### ISSUE TITLE
custom item frame renderer causes maps to have z-fighting with item frames

### ISSUE THREAD

**Issue by AlexanderwithaA** [FxMorin/MoreCulling#199]
when the custom item frame renderer option is enabled maps have z-fighting.
map from hypixel skyblock.
![image](https://github.com/FxMorin/MoreCulling/assets/101488091/c1771d8b-6b98-48b1-a035-cba88bf2c5e5)

**Comment by FxMorin** [comment]
How far are you from the maps?

**Comment by AlexanderwithaA** [comment]
about 15 blocks. I can't easily test this again.

**Comment by FxMorin** [comment]
I believe I might have fixed this at some point

**Comment by FxMorin** [comment]
I've tweaked it some more. It will be in the next release

**Comment by FxMorin** [comment]
If you get this issue again, re-open the issue. However it should be fixed in v0.21.0

**Comment by AlexanderwithaA** [comment]
@FxMorin I don't think I have the ability to re-open

**Comment by AlexanderwithaA** [comment]
Yeah, still got the issue. Tested with latest versions of mod menu/cloth config/more culling/fabric api/fabric loader

![image](https://github.com/FxMorin/MoreCulling/assets/101488091/d87109f5-f62b-4356-aabc-629f6ff15ce7)

With the custom item frame renderer

![image](https://github.com/FxMorin/MoreCulling/assets/101488091/a2f32a2b-617b-40fa-aafa-a67a6c2bf3ba)

**Comment by seriousfreezing** [comment]
Well, I've just tested it on Hypixel on the same map. And there doesn't seem to be this problem any more.
No, I'm not using any extra mods apart from MoreCulling, no extra resourcepack or anything.

**Comment by FxMorin** [comment]
Good enough for me
